scripts/barclamp_mgmt_lib.rb

```diff
@@ -38,21 +38,15 @@
 
 if ENV["CROWBAR_DIR"]
   BASE_PATH = ENV["CROWBAR_DIR"]
-  BARCLAMP_PATH = File.join BASE_PATH, 'barclamps'
-  CROWBAR_PATH = File.join BASE_PATH, 'crowbar_framework'
-  MODEL_SOURCE = File.join CROWBAR_PATH, 'barclamp_model'
-  BIN_PATH = File.join BASE_PATH, 'bin'
-  UPDATE_PATH = '/updates'
-  ROOT_PATH = '/'
 else
-  BASE_PATH = File.join '/opt', 'dell'
-  BARCLAMP_PATH = File.join BASE_PATH, 'barclamps'
-  CROWBAR_PATH = File.join BASE_PATH, 'crowbar_framework'
-  MODEL_SOURCE = File.join CROWBAR_PATH, 'barclamp_model'
-  BIN_PATH = File.join BASE_PATH, 'bin'
-  UPDATE_PATH = '/updates'
-  ROOT_PATH = '/'
+  BASE_PATH = File.join "/opt", "dell"
 end
+BARCLAMP_PATH = File.join BASE_PATH, "barclamps"
```

Comment:
I you want to make the refactoring perfect then it would be nice if you can add (). e.g `File.join(BASE_PATH, "barclamps")` hound is not able to detect this.